### PR TITLE
Init models

### PR DESCRIPTION
**Pull request by kagoromo**

### CODE REVIEW

File: app/models/admin.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Admin < ActiveRecord::Base
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```
Comment (nguyenngocha) [inline app/models/admin.rb]: ident

File: config/environments/development.rb [hunk 2]
```diff
@@ -16,6 +16,8 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (nguyenngocha) [inline config/environments/development.rb]: double quote

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Devise.setup do |config|
+  config.secret_key = '78cda59accaa7fe9a4c3ddf6ad38237d2c3679f39de625ef69579ead9dc37c7d0ea98c23db6733351f638791eadfb9d6a4bef230525ee15798bde4a4929f369c'
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (nguyenngocha) [inline config/initializers/devise.rb]: double quote

File: db/migrate/20160713042742_create_questions.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.string :content
+      t.integer :question_type
+      t.boolean :is_approved
```
Comment (nguyenngocha) [inline db/migrate/20160713042742_create_questions.rb]: default

File: db/migrate/20160713042813_create_answers.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration
+  def change
+    create_table :answers do |t|
+      t.string :content
+      t.boolean :is_correct
```
Comment (nguyenngocha) [inline db/migrate/20160713042813_create_answers.rb]: default: false

File: db/migrate/20160713042937_create_exams.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration
+  def change
+    create_table :exams do |t|
+      t.integer :status
+      t.time :spent_time
+      t.integer :score
```
Comment (nguyenngocha) [inline db/migrate/20160713042937_create_exams.rb]: default: 0

File: db/migrate/20160713043014_create_results.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+class CreateResults < ActiveRecord::Migration
+  def change
+    create_table :results do |t|
+      t.boolean :is_correct
```
Comment (nguyenngocha) [inline db/migrate/20160713043014_create_results.rb]: default: false

File: config/environments/development.rb [hunk 8]
```diff
@@ -16,6 +16,8 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
```
Comment (nguyenngocha) [inline config/environments/development.rb]: {host: "localhost", port: 3000}

File: db/migrate/20160713042742_create_questions.rb [hunk 9]
```diff
@@ -0,0 +1,12 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.string :content
+      t.integer :question_type
```
Comment (nguyenngocha) [inline db/migrate/20160713042742_create_questions.rb]: default: 0

File: db/migrate/20160713042937_create_exams.rb [hunk 10]
```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration
+  def change
+    create_table :exams do |t|
+      t.integer :status
```
Comment (nguyenngocha) [inline db/migrate/20160713042937_create_exams.rb]: default: 0

**Comment by nguyenngocha** [comment]
OK

**Comment by tranduytung** [comment]
OK

**Comment by daonhat** [comment]
OK

**Comment by duccm94** [comment]
OK

**Pull request merged**